homework/index.js

```diff
@@ -30,13 +30,108 @@
     return elem;
   }
 
+  function update(val, dBase) {
+    const container = document.getElementById('container');
+    const leftDiv = createAndAppend('div', container, { class: 'left-div whiteframe' });
+    const table = createAndAppend('table', leftDiv);
+    const tbody = createAndAppend('tbody', table);
+    const rep = dBase[val];
+
+    for (let ix = 0; ix < 4; ix++) {
+      const row = createAndAppend('tr', tbody);
+      const tdFirst = createAndAppend('td', row, { class: 'label' });
+      const tdSec = createAndAppend('td', row);
+      const updateDate = new Date(rep.updated_at).toLocaleString();
+      // eslint-disable-next-line default-case
+      switch (ix) {
+        case 0:
+          tdFirst.textContent = 'Repository :';
+          createAndAppend('a', tdSec, { text: rep.name, href: rep.html_url });
+          break;
+        case 1:
+          tdFirst.textContent = 'Description :';
+          tdSec.textContent = rep.description;
+          break;
+        case 2:
+          tdFirst.textContent = 'Forks :';
+          tdSec.textContent = rep.forks;
+          break;
+        case 3:
+          tdFirst.textContent = 'Updated :';
+          tdSec.textContent = updateDate;
+          break;
+      }
+    }
+
+    const rightDiv = createAndAppend('div', container, { class: 'right-div whiteframe' });
+    createAndAppend('p', rightDiv, {
+      text: 'Contributions',
+      class: 'contributor-header',
+    });
+    const ul = createAndAppend('ul', rightDiv, { class: 'contributor-list' });
+    const urlCol = rep.contributors_url;
```

Comment:
What does `Col` mean in `urlCol`? I don't get it. Why do you need this variable anyway? Why not just do:

```js
fetchJSON(rep.contributors_url, (error, contributors) => {
```

Avoid cryptic names such as `e` and `dt` unless it is absolutely clear (not just to yourself) what they stand for.

Reply:
I was thinking "Collaborators" while i write urlCol. I have forgotten "Contributors". 
Got it I am going to avoid cryptic names.